server/cmake/modules/FindLibClang.cmake

```diff
@@ -61,20 +61,20 @@ endforeach()
 
 find_path(LIBCLANG_INCLUDE_DIR clang-c/Index.h
   PATHS ${libclang_llvm_header_search_paths}
-  PATH_SUFFIXES LLVM/include #Windows package from http://llvm.org/releases/
+  PATH_SUFFIXES llvm/include #Windows package from http://llvm.org/releases/
```

Comment:
Sorry but would you mind telling me what package you installed? I downloaded 2 packages from http://llvm.org/releases/download.html:

1. clang+llvm-3.7.0-amd64-unknown-freebsd10
2. clang+llvm-3.7.0-x86_64-apple-darwin

None of them provide a path such as `llvm/include`.

If you installed one of this package a decided to put it in `/usr/local/llvm` that's not standard, not more standard than putting it in `/opt/my-llvm-install/`.

In this case, you have to tell CMake about your non-standard path with `-DCMAKE_PREFIX_PATH=/opt/my-llvm-install`.

On Windows, the packages installs in `C:/Program File/LLVM/` I think, and I used the PATH_SUFFIXES for that, even though I'm am not sure it is the proper way to do but the PATH_SUFFIXES.